core/services/job/orm.go

```diff
@@ -403,6 +403,7 @@ func (o *orm) JobsV2() ([]Job, error) {
 	var jobs []Job
 	err := postgres.GormTransactionWithDefaultContext(o.db, func(tx *gorm.DB) error {
 		err := PreloadAllJobTypes(tx).
+			Preload("JobSpecErrors").
```

Comment:
I have a similar change here (together with some more API changes): https://github.com/smartcontractkit/chainlink/pull/4715/commits/a93a798f47f6cf76873e67b55cafe864bb78fe09#diff-c15a74a1728815973c97643627390591c30f41dc0501779bc30e664ffa137fe5R417

FindJob also needed the preload.

Reply:
I've removed it. from here. I just used it to test that errors were properly showing